include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -209,6 +210,10 @@ table KUKAiiwaArmConfiguration {
   /// position, cartesian impedence, or joint impedence low level controller adjustments
   controlMode:EControlMode;
 
+  setCartImpedance:CartesianImpedenceControlMode;
```

Comment:
Specific configuration may be best stored in an equivalent to ArmState for configuration... looking for the right line to see if something already exists.